ecommerce/api.py

```diff
@@ -580,6 +581,22 @@ def import_wire_transfer(wire_transfer, header_row):
             },
             order=order,
         )
+
+        if application.is_paid_in_full:
+            application.state = AppStates.COMPLETE.value
+            application.save()
+
+            _, created = BootcampRunEnrollment.objects.get_or_create(
+                user=user, bootcamp_run=bootcamp_run
+            )
```

Comment:
Might be better to replace the above with 

```python
complete_successful_order(order, send_receipt=????)
``` 

Not sure if we'd want to send a receipt to the user or not, Peter probably knows.
The above function would also run `novoed_tasks.enroll_users_in_novoed_course.delay` if appropriate.